lib/child_process.js

```diff
@@ -195,6 +195,8 @@ exports.execFile = function(file /*, args, options, callback*/) {
 
   var ex = null;
 
+  var cmd = file;
+
```

Comment:
Well, if this is not fixed on the V8 side, I can try to check the libs for these cases.